package.json

```diff
@@ -28,16 +28,18 @@
   "dependencies": {
     "@amplitude/analytics-browser": "^0.3.1",
     "@directus/sdk": "^10.1.4",
-    "@mantine/core": "^4.2.2",
-    "@mantine/hooks": "^4.2.2",
-    "@mantine/ssr": "^4.2.2",
-    "@pokt-foundation/pocket-blocks": "1.0.21",
+    "@emotion/react": "^11.10.4",
```

Comment:
Not needed now, but these should probably be part of pocket-blocks and we shouldnt have to have them as dependencies for the portal.

Reply:
Agree with you.